### ISSUE TITLE
MsgPack.dll fails Windows App Certification Validation

### ISSUE THREAD

**Issue by benjamin-bader** [msgpack/msgpack-cli#65]
Microsoft has a list of approved native API methods they will allow for apps to be published in their app store; memcmp is not one of them.

An app using msgpack-cli as of today will fail validation with the following error:
```
This API is not supported for this application type - Api=memcmp. Module=msvcrt.dll. File=MsgPack.dll.
```

WinRT, at least the Store flavors, should use the slow-compare path.

**Comment by yfakariya** [comment]
Sure. I'll change it.

**Comment by yfakariya** [comment]
I fixed this issue in 0.5.10-beta1 with issue #64.

**Comment by benjamin-bader** [comment]
Thanks!
